systemd/amd64/jessie/entry.sh

```diff
@@ -28,7 +46,9 @@ if [ "$INITSYSTEM" = "on" ]; then
 	echo -e "${GREEN}Systemd init system enabled."
 	env > /etc/docker.env
 
-	echo -e "#!/bin/bash\n exec $@" > /etc/resinApp.sh
```

Comment:
Why don't we just replace this with

```bash
printf '#!/bin/bash\n exec ' > /etc/resinApp.sh
printf '%q ' "$0" "$@" > /etc/resinApp.sh
```

instead of doing all the above?